plugins/module_utils/nae.py

```diff
@@ -1053,6 +1056,9 @@ def new_object_selector(self):
             self.result['Result'] = final_msg
 
     def new_traffic_selector(self):
+        obj = self.query_compliance_object(json.loads(self.params.get('form')).get('name'))
```

Comment:
You might want to do a try/except around the json.loads in case it is not a proper JSON.
You might also want to verify that the return of json.loads is not None to be able to do a get.